File: bj/2920.py
import sys
notes = list(sys.stdin.readline().strip().split())
notes = [int(_) for _ in notes]

direction = -1
for i in range(len(notes)-1):
    if i == 0:
        if notes[i] < notes[i+1]:
            direction = 1
    else:
        if (notes[i+1] - notes[i])*direction < 0:
            direction = 0
            break

if direction == 1:
    print("ascending")
elif direction == -1:
    print("descending")
else:
    print("mixed")

File: bj/2920.cpp
#include<iostream>
#include<vector>
using namespace std;

int main(){
    int direction = -1;
    vector<int> notes(8);

    for (int i = 0; i < 8; i++){
        cin >> notes[i];
    }

    for (int j = 0; j < 7; j++){
        if (j == 0){
            if (notes[j] < notes[j+1]){
                direction = 1;
            }
        }
        else{
            if ((notes[j+1]-notes[j])*direction < 0){
                direction = 0;
                break;
            }            
        }
    }

    if (direction == 1){
        cout << "ascending";
    }
    else if (direction == -1){
        cout << "descending";
    }
    else{
        cout << "mixed";
    }

    return 0;
}